packages/transport-dom/src/create.ts

```diff
@@ -49,8 +49,8 @@ export interface ChannelTransportOptions
 
 export const createChannelTransport = ({
   getPort,
-  defaultTimeoutMs,
   jsonOptions,
+  defaultTimeoutMs = 10_000,
```

Comment:
applied a default here, instead of using a shared configuration object repo-wide. this seems more consumeable